platform/ariane-fpga/config.mk

```diff
@@ -0,0 +1,44 @@
+#
+# SPDX-License-Identifier: GPL-2.0
+#
+# Copyright (C) 2019 FORTH-ICS/CARV
+#		Panagiotis Peristerakis <perister@ics.forth.gr>
+#
+
+#for more infos, check out /platform/template/config.mk
+
+PLATFORM_RISCV_XLEN = 64
+
+# Common drivers to enable
+PLATFORM_SERIAL_UART8250=y
+PLATFORM_IRQCHIP_PLIC=y
+PLATFORM_SYS_CLINT=y
+
+# Blobs to build
+FW_TEXT_START=0x80000000
+FW_JUMP=n
+
+ifeq ($(PLATFORM_RISCV_XLEN), 32)
+ # This needs to be 4MB aligned for 32-bit support
+ FW_JUMP_ADDR=0x80400000
+ else
+ # This needs to be 2MB aligned for 64-bit support
+ FW_JUMP_ADDR=0x80200000
+ endif
+FW_JUMP_FDT_ADDR=0x82200000
+
+# Firmware with payload configuration.
+FW_PAYLOAD=y
+
+ifeq ($(PLATFORM_RISCV_XLEN), 32)
+# This needs to be 4MB aligned for 32-bit support
+  FW_PAYLOAD_OFFSET=0x400000
+else
+# This needs to be 2MB aligned for 64-bit support
+  FW_PAYLOAD_OFFSET=0x200000
+endif
+FW_PAYLOAD_FDT_ADDR=0x82200000
+FW_PAYLOAD_ALIGN=0x1000
+
+PLATFORM_INCLUDE_LIBC=y
+PLATFORM_INCLUDE_LIBFDT=y
```

Comment:
Can you rebase your patch on master ? We won't need LIBC & LIBFDT anymore as they will be part of libsbiutils.a

http://lists.infradead.org/pipermail/opensbi/2019-June/000209.html